LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [2021] [IOG]
+   Copyright [2022] [IOG]
```

Comment:
Shouldn't this be a range? According to this SO answer and various other resources on the web( https://opensource.stackexchange.com/questions/6389/how-do-the-years-specified-in-a-copyright-statement-work) it seems to be "best practice" for work that have multiple releases or publication dates, like software, to include a range of years starting with initial publication year.

Reply:
Of course, I am not a lawyer so 🤷

Reply:
Yeah, you're probably right. Not changing it or making it a range is probably better.